test/integration/home_page_test.rb

```diff
@@ -0,0 +1,9 @@
+require 'test_helper'
+
+class HomePageTest < ActionDispatch::IntegrationTest
+  test "visitor lands on homepage with slider" do
+    visit '/'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.